mkdocs.yaml

```
@@ -186,3 +186,47 @@ markdown_extensions:
       emoji_generator: !!python/name:material.extensions.emoji.to_svg
   - pymdownx.arithmatex:
       generic: true
+
+nav:
```

YAMLlint (1.37.1)

[error] duplication of key "nav" in mapping

(key-duplicates)